src/App.jsx

```diff
@@ -14,57 +15,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => setSelectedGood('')}
+          />
+        )}
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      <table className="table">
+        <TBody selectedGood={selectedGood} setSelectedGood={setSelectedGood} />
+      </table>
+    </main>
+  );
+};
 
-        <tr data-cy="Good" className="has-background-success-light">
+const TBody = ({ selectedGood, setSelectedGood }) => {
+  return (
+    <tbody>
+      {goods.map(good => (
+        <tr
+          key={good}
+          data-cy="Good"
+          className={
+            good === selectedGood ? 'has-background-success-light' : ''
+          }
```

Comment:
Consider simplifying the `className` assignment by using a template literal or a conditional (ternary) operator directly within the `className` attribute. This can enhance readability and maintainability of the code.